presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -407,6 +408,12 @@ public FeaturesConfig setSpillerSpillPaths(String spillPaths)
         return this;
     }
 
+    @AssertTrue(message = "experimental.spiller-spill-path must be configured when experimental.spill-enabled is set to true")
```

Comment:
Somewhat hacky. Is this approach used in other places?

Test?

Reply:
No, it's not used elsewhere. Do you have a better idea?

We don't have tests that validation in `FeaturesConfig` works. I added such a test nonetheless.